gcloud/bigtable/happybase/connection.py

```diff
@@ -18,8 +18,15 @@
 import datetime
 import warnings
 
+from grpc.beta import interfaces
+from grpc.framework.interfaces.face import face
 import six
 
+try:
+    from happybase.hbase.ttypes import AlreadyExists
+except ImportError:
+    from gcloud.exceptions import Conflict as AlreadyExists
```

Comment:
@tseaver I didn't add a `# pragma: NO COVER` since the `happybase.hbase` import fails in `tox`, hence both branches are always covered.